src/drf_yasg/generators.py

```diff
@@ -34,15 +34,28 @@ def get_path_from_regex(self, path_regex):
             logger.warning("url pattern does not end in $ ('%s') - unexpected things might happen", path_regex)
         return self.unescape_path(super(EndpointEnumerator, self).get_path_from_regex(path_regex))
 
-    def should_include_endpoint(self, path, callback, app_name='', namespace='', url_name=None):
+    def get_version_regex_from_path_regex(self, path_regex, version_param):
+        if path_regex.endswith(')'):
+            logger.warning("url pattern does not end in $ ('%s') - unexpected things might happen", path_regex)
+        version_param_regex = re.compile(rf"\(\?P<{version_param}>(?P<value>.+?)\)")
```

Comment:
You can't use an f-string here since we still support Python 2.7 and 3.5.

Reply:
Fixed, rolling back to the good old `%` operator.